docs/source/core_concepts/glossary.rst

```diff
@@ -45,142 +36,71 @@ Signable content is called payload, so the structure of a block looks like this:
     - hash of a previous block in the chain
     - rejected transactions hashes — array of transaction hashes, which did not pass stateful validation step; this field is optional
 
-Block Creator
-=============
-
-System component that forms a block from a set of transactions that
-have been passed `stateless <#stateless-validation>`__ and `stateful <#stateful-validation>`__
-validation for further propagation to `consensus <#consensus>`__.
 
 Client
 ======
 
 Any application that uses Iroha is treated as a client.
 
-A distinctive feature of Iroha is that all clients are using simple client-server abstractions
-when they interact with a peer network: they don't use any abstractions
-which are specific for blockchain-related systems.
-For example, in Bitcoin clients have to validate blocks,
-or in Fabric they need to poll several peers to make sure that a transaction was written in a block,
-whereas in Iroha a client interacts with any peer similarly to a single server.
+A distinctive feature of Iroha is that all clients are using simple client-server abstractions when they interact with a peer network: they don't use any abstractions which are specific for blockchain-related systems.
+For example, in Bitcoin clients have to validate blocks, or in HL Fabric they need to poll several peers to make sure that a transaction was written in a block, whereas in HL Iroha a client interacts with any peer similarly to a single server.
 
 Command
 =======
 
-A command is an intention to change the `state <#world-state-view>`__.
-For example, in order to create a new `role <#role>`__ in Iroha you have to issue
-`Create role <../api/commands.html#create-role>`__ command.
-
-Consensus
-=========
-
-A consensus algorithm is a process in computer science used to achieve agreement
```

Comment:
I suggest to leave "consensus algorithm" and "Consensus, as an algorithm" description here, since it is general.